app/src/main/java/org/mozilla/fenix/search/SearchDialogFragment.kt

```diff
@@ -382,11 +382,9 @@ class SearchDialogFragment : AppCompatDialogFragment(), UserInteractionHandler {
             // We delay querying the clipboard by posting this code to the main thread message queue,
             // because ClipboardManager will return null if the does app not have input focus yet.
             lifecycleScope.launch(Dispatchers.Cached) {
-                store.dispatch(
-                    SearchFragmentAction.UpdateClipboardUrl(
-                        requireContext().components.clipboardHandler.url
-                    )
-                )
+                context?.components?.clipboardHandler?.url?.let { clipboardUrl ->
```

Comment:
NB: I am not using `runIfFragmentAttached` here because it's not thread-safe and we're not on `MAIN`.